### ISSUE TITLE
cannot see chinese text in preference

### ISSUE THREAD

**Issue by luboq** [rhiokim/haroopad#505]
have to switch to english gui by deleting zh-CN.pak to make it normal. 
fresh new install, v0.13.1, win7 x64.

---

![2](https://cloud.githubusercontent.com/assets/3130930/7442215/5829aa80-f13a-11e4-830d-662995d5d219.gif)

**Comment by rhiokim** [comment]
link: http://haroopad.userecho.com/topic/711841-chinese-characters-dont-display-in-the-edit-zone/

**Comment by jack1582** [comment]
I have the same problem. Both edit panel and preference panel .

**Comment by LanderlYoung** [comment]
I had the same experience, and i **FIXED** it in this way(this helps in ther editor area):
1. goto `C:\Users\%user%\AppData\Local\Haroopad\Themes\editor`
2. add this `font-family: "微软雅黑", "Meiryo UI", "Malgun Gothic", "Segoe UI", "Trebuchet MS", Helvetica, sans-serif !important;` to editor{} closure.
3. well, at last reatart your haroop pad

have fun!

Obviously, this problem is caused by some kind of a font problem, and change the font setting will solve part of it.

**Comment by rhiokim** [comment]
@LanderlYoung Thanks a lot. It'll be fixed with next release.

**Comment by rhiokim** [comment]
Hi there

I've made new version package for next release. it's release candidate version, not official version
https://bitbucket.org/rhiokim/haroopad-download/downloads

I had tried to solve this issue many time. But I did not solve yet. Because I can't get same environment. When I changed my system local from Korea to China. At this time, it's work fine. 

Could I get your feedback about this, If you have spare time for Haroopad?
Thanks a lot.
